packages/app/src/device-info/presenters/device-info.presenter.ts

```diff
@@ -32,10 +47,11 @@ export class DeviceInfoPresenter {
       memorySpace: {
         reservedSpace:
           fromMebiToByte(Number(data.systemReservedSpace)) +
-          missingStorageBytes,
+          getMissingStorageBytes(Number(data.deviceSpaceTotal), deviceType),
         usedUserSpace: fromMebiToByte(Number(data.usedUserSpace)),
         total:
-          fromMebiToByte(Number(data.deviceSpaceTotal)) + missingStorageBytes,
+          fromMebiToByte(Number(data.deviceSpaceTotal)) +
+          getMissingStorageBytes(Number(data.deviceSpaceTotal), deviceType),
       },
```

Comment:
`getMissingStorageBytes(Number(data.deviceSpaceTotal), deviceType)` is executed twice, maybe invoke function, and store this to variable before return?